src/Json.Schema.ToDotNet.Cli/Json.Schema.ToDotNet.Cli.nuspec

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.Json.Schema.ToDotNet</id>
+    <version>$version$</version>
+    <title>Microsoft Json Schema to .NET Object Model Generator</title>
+    <authors>microsoft</authors>
+    <owners>microsoft,tse-securitytools</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>A utility that generates a .NET object model from a JSON schema.</description>
+    <releaseNotes>Version $version$ of the JSON .NET object model generator (for SARIF v2.0.0)</releaseNotes>
```

Comment:
>for SARIF v2.0.0 [](start = 77, length = 16)

There is nothing SARIF v2.0-specific about this tool. It will generate an object model for any schema (with certain limitations on the JSON schema constructs it can handle). It just happens that in building the SARIF SDK, we run it against the SARIF v2.0 schema.